x-pack/plugins/enterprise_search/public/applications/workplace_search/routes.ts

```diff
@@ -72,7 +72,7 @@ export const ADD_ONEDRIVE_PATH = `${SOURCES_PATH}/add/one-drive`;
 export const ADD_SALESFORCE_PATH = `${SOURCES_PATH}/add/salesforce`;
 export const ADD_SALESFORCE_SANDBOX_PATH = `${SOURCES_PATH}/add/salesforce-sandbox`;
 export const ADD_SERVICENOW_PATH = `${SOURCES_PATH}/add/service-now`;
-export const ADD_SHARE_POINT_PATH = `${SOURCES_PATH}/add/share-point`;
+export const ADD_SHAREPOINT_PATH = `${SOURCES_PATH}/add/share-point`;
```

Comment:
Definitely not for this PR, but we should probably also change these paths before 8.0 (service-now -> servicenow)

Reply:
https://github.com/elastic/kibana/pull/83459/commits/2fa215f13debb7cc00755569e313a1504219d748